DirectProgramming/DPC++/DenseLinearAlgebra/simple-add/src/simple-add-buffers.cpp

```diff
@@ -23,8 +23,17 @@
 #include <array>
 #include <iostream>
 #include "dpc_common.hpp"
```

Comment:
Please add the comments below for for the include of dpc_common.hp. it should look like this:

// dpc_common.hpp can be found in the dev-utilities include folder.
// e.g., $ONEAPI_ROOT/dev-utilities/<version>/include/dpc_common.hpp
#include "dpc_common.hpp"